### PR TITLE
Settings flow

### PR DESCRIPTION
**Pull request by evie-adenoyin**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import 'react-toastify/dist/ReactToastify.css'
```
Comment (slimsolz) [inline src/App.js:3]: Please pull the latest and make the necessary changes to this file

Comment (slimsolz) [reply src/App.js:3]: I believe that should also fix the conflicts

Comment (evie-adenoyin) [reply src/App.js:3]: Okay

**Comment by slimsolz** [comment]
LGTM

**Pull request merged**